doc/api/webstreams.md

```diff
@@ -1694,6 +1694,41 @@ buffer(readable).then((data) => {
 });
 ```
 
+#### `streamConsumers.bytes(stream)`
+
+<!-- YAML
+added: v25.0.0
+-->
+
+* `stream` {ReadableStream|stream.Readable|AsyncIterator}
+* Returns: {Promise} Fulfills with a {Uint8Array} containing the full
+  contents of the stream.
+
+```mjs
+import { bytes } from 'node:stream/consumers';
+import { Readable } from 'node:stream';
+
+const dataBuffer = Buffer.from('hello world from consumers!');
```

Comment:
It looks like lint-md is complaining about this line – you'll need to add an explicit Buffer import from `node:buffer` to your examples to make it happy!